packages/preview-middleware-client/types/sap.ui.rta.d.ts

```diff
@@ -106,6 +106,7 @@ declare module 'sap/ui/rta/command/OutlineService' {
         instanceName?: string;
         name?: string;
         icon?: string;
+        component?: boolean;
```

Comment:
Can this appear in all of the different types of outline node types (`aggregation`, `element` and `extensionPoint`)?  If it only is present in specific node types, it should be added only for those interfaces.

Reply:
I'm not quite sure about that, that's why I put it in the base interface.